src/pages/Room/index.js

```diff
@@ -37,30 +37,18 @@ const Body = styled.div`
   background-color: #e0dede;
 `;
 
-const Header = styled.div`
-  display: flex;
-  position: relative;
-  width: 100%;
-  justify-content: center;
-  padding-bottom: 30px;
-  margin-bottom: 30px;
-  border-bottom: 1px solid #000;
-  h2 {
-    display: inline-block;
-    font-size: 2em;
-  }
-  .backBtn {
-    position: absolute;
-    left: 0;
-    top: 50%;
-    padding: 10px;
-    margin-top: -35px;
-    background-color: inherit;
-    font-size: 25px;
-  }
+const BackBtn = styled.button`
```

Comment:
GoBack 같은 이름으로도 네이밍을 할 수 있을거 같아서 이것도 한번 추천드려봅니다. 언제나 네이밍 짓기가 어려운거 같아요 ㅜㅜ 
```suggestion
const BackButton = styled.button`
```

Reply:
바꾸겠습니다!